File: desafio.js
// Desafios JavaScript na DIO tem funções "gets" e "print" acessíveis globalmente:
// - "gets": lê uma linha com dados de entrada (input) do usuário;
// - "print": imprime um texto de saída (output) pulando uma linha.
// - "gets" e "print" não são funções nativas da linguagem e sim criadas para os desafios na DIO.

// Lê os valores de entrada:
const valorSalario = parseFloat(gets());
const valorBeneficios = parseFloat(gets());

// Calcula o imposto através da função calcularImposto():
const valorImposto = calcularImposto(valorSalario);

// Calcula e imprime a saída (com 2 casas decimais):
const saida = (valorSalario - valorImposto) + valorBeneficios;
print(saida.toFixed(2));

// Função para o cálculo do imposto (baseado nas alíquotas):
function calcularImposto(salario){
    let aliquota;
    if (salario >= 0 && salario <= 1100){
        aliquota = 0.05;
    }
    else if (valorSalario > 1100 && valorSalario <= 2500){
        aliquota = 0.10;
    } else{
        aliquota = 0.15;
    }    
    return aliquota * salario;
}

File: desafio.py
'''
Para ler e escrever dados em Python, utilizamos as seguintes funções:
- input: lê uma linha com dados de entrada (input) do usuário;
- print: imprime um texto de saída (output) pulando uma linha.
'''

# Função para o cálculo do imposto (baseado nas alíquotas):
def calcular_imposto(salario):
    aliquota = 0.00
    if(salario >= 0 and salario <= 1100):
        aliquota = 0.05
    # TODO Criar as demais condições para as alíquotas de 10,00% e 15,00%.
    return aliquota * salario

# Lê os valores de entrada:
valor_salario = float(input())
valor_beneficios = float(input())

# Calcula o imposto através da função calcular_imposto():
valor_imposto = calcular_imposto(valor_salario)

# Calcula e imprime a saída (com 2 casas decimais):
saida = (valor_salario - valor_imposto) + valor_beneficios
print(f'{saida:.2f}')